src/components/views/Leaderboard.tsx

```diff
@@ -33,6 +34,9 @@ const factionOptions: SelectButtonOption[] = [
   { label: "Vanguard", value: Race.VANGUARD, icon: vanguard.src },
 ]
 
+const orderButtonHighlighedClasses = classes(styles.button.base, styles.button.highlighted, "px-3 py-1.5")
```

Comment:
When using the `styles.button.set` you can style the inside buttons with just `styles.button.sm` for the spacing. Similar to how the search input is styles

Reply:
This will also improve the text size

Reply:
Good call